nautobot/extras/tests/test_api.py

```diff
@@ -975,6 +975,7 @@ def setUpTestData(cls):
         )
 
         for query in graphqlqueries:
+            query.clean()
```

Comment:
Can you make this `full_clean()` instead, please? That is more correct. (We still need to refactor the places `clean()` is used over `full_clean()` but might as well make sure new code uses the latter).